contracts/eosio.system/include/eosio.system/eosio.system.hpp

```diff
@@ -610,21 +610,21 @@ namespace eosiosystem {
       time_point_sec utilization_timestamp   = {};                 // When adjusted_utilization was last updated
    };
 
-   struct [[eosio::table("rent.state"),eosio::contract("eosio.system")]] rentbw_state {
-      static constexpr uint32_t default_rent_days = 30; // 30 day resource rentals
+   struct [[eosio::table("powup.state"),eosio::contract("eosio.system")]] powerup_state {
```

Comment:
Renaming tables will break testnets which deployed rentbw

Reply:
The new `powup.state` table will be inconsistent with the other resource-related tables and the weights already assigned to existing accounts which have outstanding rents.

Reply:
Is there an option to deploy a modified `rentbw-1.9` contract that disables renting and wait for orders to all expire and then deploy the changed contract? It is not ideal but I believe that would work to migrate.

Reply:
Never mind that would effectively stop all traffic on the network for 30 days.

Reply:
The tests could run a patched version which renames the tables back, but that would create a noticeable difference to tooling which runs on both test nets and non-test nets.

Reply:
A modified `rentbw-1.9` could disable new rentals and allow `rentbwexec` to expire existing rentals prematurely. An account with unlimited cpu & net would have to manage this. All normal accounts would be unusable because of `eosio.reserv` until `powerup-1.9` is installed and activated.

Reply:
Actually, the modified version needs to clear out the `eosio.reserv` allocation or `powerup-1.9` will malfunction.